### PR TITLE
Undraft the bylaws: CF Council [due: 7 May]

### PR DESCRIPTION
**Pull request by ebullient**
[🗳️ Vote progress](https://github.com/commonhaus/foundation/pull/149#issuecomment-2087745125 "IC_kwDOKRPTI858cHZl")

Just the CF Council (and Extended Governance Committee) bylaws this time.

I know I have a concern about electronic/digital participation in lieu of meetings. At the moment (as written) our electronic participation threshold (quorum) is everyone. Not sure how that will scale as we add more projects. 

I think Gavin also noticed that it wasn't necessarily clear if procedures were the same or different for bylaws vs. policies. 

voting group: @commonhaus/cf-egc

Do one of the following:  

- Approve the PR or react with **👍 (:+1:)** if it looks good to you
- Review with Comments or react with **👀 (:eyes:)** if you're "ok" with it (it may not be your favorite)
- If you think it needs discussion or revision
    - Create a review, add your comments and require changes
    - Use the +- button to make a suggestion (instead of just adding a comment).

### CODE REVIEW

**Review by kenfinnigan — APPROVED** [review]

**Review by kenfinnigan — APPROVED** [review]
Re-reviewed after updates.

Agree with all the changes

**Review by timtebeek — APPROVED** [review]

**Review by aalmiray — APPROVED** [review]

**Review by evanchooly — APPROVED** [review]

File: bylaws/3-cf-council.md [hunk 1]
```diff
@@ -169,14 +169,14 @@ When consensus cannot be reached, voting is utilized to resolve differences.
 
 ### Matters approved by Supermajority Vote of the EGC
 
-- Amending the Articles of Incorporation or the Bylaws.
+- Amending the Articles of Incorporation, Bylaws or Policies.
```
[the remark this answers is not in the document]
Comment (ebullient) [reply bylaws/3-cf-council.md]: Ask and ye shall receive. OTOH, you totally could have made that change so I could just accept it. ;)

**Review by cowtowncoder — APPROVED** [review]

**Review by cealsair — APPROVED** [review]

**Review by gavinking — APPROVED** [review]
I'm approving,  but it seems to me there's a real problem with the structure of this document: IMO it fails to sufficiently-clearly separate the roles of and processes followed by the CFC vs the EGC.

**Comment by ebullient** [comment]
> I'm approving, but it seems to me there's a real problem with the structure of this document: IMO it fails to sufficiently-clearly separate the roles of and processes followed by the CFC vs the EGC.

Open a discussion so we can chase what should be clarified?

What I will add: 

- You don't want to over-specify processes in the bylaws. The general process is the same for the CFC and the EGC.
- The essential part for this section is Council (board) composition and how foundation leadership decisions are made, specifically: what can be done electronically vs. what must be done in a meeting, and what threshold of participation (quorum) and of consensus applies to which types of decisions.

**Comment by gavinking** [comment]
> Open a discussion so we can chase what should be clarified?

sure, we can figure it out next week. no need to block this PR.

**Review by maxandersen — APPROVED** [review]

**Pull request merged**